src/main/java/worldcities/ParseWorldCities.java

```diff
@@ -0,0 +1,59 @@
+package worldcities;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ParseWorldCities {
+    private double lon;
+    private double lat;
+
+    private List<City> cities;
+
+    public ParseWorldCities() throws IOException {
+        cities = loadCityListFromCsv();
+    }
+
+    public City findClosestCity(List<City> cities, double lat, double lon) {
```

Comment:
Why are you passing `cities` into this method? It's already available in this class.